### ISSUE TITLE
Allow multiple cannot be deselected

### ISSUE THREAD

**Issue by PhilippSelenium** [chocolatey/chocolatey-ansible#35]
this playbook task
```
- name: "Uninstall python version {{ uninstall_version }}"
    chocolatey.chocolatey.win_chocolatey:
      name: python3
      version: 3.6.8
      state: absent
      allow_multiple: false
      params: "/InstallDir:\\Python36"
```

results in the following command:
```
"command": "C:\\ProgramData\\chocolatey\\bin\\choco.exe uninstall python3 --yes --no-progress --limit-output --package-params /InstallDir:C:\\Python36 --timeout 2700 --allow-multiple --version 3.6.8"
```
1. Even though I explicitly deselected ```allow-multiple``` the command is executed with this flag
2. The default of ```allow-multiple``` is supposed to be ```false```: https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/win_chocolatey_module.html

ansible-version: 2.10.7
choco version : 0.10.5

**Comment by vexx32** [comment]
Not sure who wrote / maintains those docs, but yeah the --allow-multiple thing is all over this module, and I don't recall seeing any code that actually looks for an option to disable it. Probably the default should be false, as it can cause other issues.

Thanks for reporting!

**Comment by PhilippSelenium** [comment]
@vexx32 Just to be clear. My issue is not with the docs. It is that I cannot uninstall python3 via ansible because the ```--allow_multiple``` flag is set no matter what I do.

**Comment by vexx32** [comment]
You should be able to uninstall it if you remove the `version:` parameter, I think?

**Comment by PhilippSelenium** [comment]
@vexx32 Okay I will try that thanks, the problem is that I have more than one python version installed and want to explicitly remove that one.

Can I hope for a fix in the near future or should I go for a workaround?

**Comment by PhilippSelenium** [comment]
Sorry, I just found that https://github.com/chocolatey/chocolatey-ansible/pull/32 with a pending review for 2 months

**Comment by vexx32** [comment]
No worries. Yeah, ran into some weird errors doing that and haven't had a chance to have a close look. I'll have another look today and see if I can't get that sorted. That's the main issue with --allow-multiple being everywhere, but at some point we will need to actually hook up the parameter properly and make sure it's properly configurable.

**Comment by vexx32** [comment]
I think we can consider this fixed by #32. Leaving `--allow-multiple` in the hands of an individual invocation is hazardous at the moment because of the broken behaviour in `choco` (see #21)

The behaviour for `state: absent` as of #32 is:

1. Ignore user-provided `allow_multiple`, and
2. If a specific version is specified:
   a.  If there are multiple package versions installed, provide `--allow-multiple` so that only the one target version can be installed.
   b. If there is only one version installed, do not provide `--allow-multiple`, `choco` will break if it is provided in this instance.
3. Otherwise, simply call `choco uninstall` without `--allow-multiple`; all versions will be uninstalled.

For 2b, I'm realizing there's an edge case I need to account for -- `state: absent` with a specific `version` that _doesn't_ match the single installed version. If this happens, we need to no-op.
